napari/_qt/qt_event_loop.py

```diff
@@ -176,6 +181,8 @@ def quit_app():
 def gui_qt(*, startup_logo=False, gui_exceptions=False, force=False):
     """Start a Qt event loop in which to run the application.
 
+    NOTE: This method will soon be deprecated. See ``run()`` instead.
```

Comment:
Are we ready to actually deprecate? It should probably be scheduled for removal in `0.5.0`? We can go one more release, but then I think can probably deprecate it in `0.4.6`.

```suggestion
    NOTE: This method will soon be deprecated. Please use ``napari.run()`` instead.
```

I have a slight preference for `Please use` over `See` as it's a little more explicit

Reply:
I'm ready to deprecate! :)  just tell me when